api/controllers/MapController.js

```diff
@@ -119,78 +465,57 @@ module.exports = {
 			var population = 0;
 			for (var i = pointGrid.length - 1; i >= 0; i--) {
 				for (var j = pointGrid[i].length - 1; j >= 0; j--) {
+					function chooseCity(location){
+						var whichType = getRandom(1, cities[k].properties.city3s+cities[k].properties.city4s+cities[k].properties.city5s);
+                		if(whichType <= cities[k].properties.city3s){
+                    		//add the city to the city3s object, which will be added to the geoJSON file at the end
+                    		city3s.push({"geometry":{"type":"Point","coordinates":pointGrid[i][j].latLng},"type":"Feature","id":cities[k].properties.name+i+j})
+                    		//add the city to the city object
+                    		city.cityParts.push({type:"city3",location:location,latLng:pointGrid[i][j].latLng,neighborhoodPop:100000});
+                    		population+=100000
```

Comment:
This should also be config, you want to keep your logic separate from your settings. then you can pull from database later, or change based on contexts.